### ISSUE TITLE
Import CSV file to table not working

### ISSUE THREAD

**Issue by bojned** [microsoft/AzureStorageExplorer#5469]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.22.1

### Regression From

1.21.3

### Architecture

x64

### Storage Explorer Build Number

20220122.3

### Platform

Windows

### OS Version

Windows 10 Pro

### Bug Description

I cannot import big CSV files to Azure tables anymore. We have an Azure table with approximately 30 columns and 33.000 records.
I can export this table just fine using storage explorer, but when I try to import it back, nothing happens. CSV file is ~30 MB in size.

If I leave storage explorer running long enough (7-8 minutes at least), then import job shows as started, but there is no track of progress at all. I've left it running for another 10 minutes, but no data gets imported and error is thrown saying that entry already exists (shouldn't overwrite be default behavior?)

This happens both with existing and new (empty) tables.

### Steps to Reproduce

1. Launch storage explorer
2. Select existing table or create new one
3. Click on import button and select CSV file

### Actual Experience

Job does not start in timely manner. No progress shown. No error messages either.
![2022-02-24_09h11_16](https://user-images.githubusercontent.com/49586535/155495012-91a22ff0-0991-42a5-97a8-a2f2db25aaed.png)
 

### Expected Experience

Import starts in reasonable time. Import progress shown

### Additional Context

On one of my VM-s, I have an older version of Storage explorer (1.21.3), and these operations work as expected:
![2022-02-24_09h51_35](https://user-images.githubusercontent.com/49586535/155500710-f8cfed1c-f770-4e5d-be63-1dc4d9079219.png)

**Comment by bojned** [comment]
[lockedout.csv](https://github.com/microsoft/AzureStorageExplorer/files/8131499/lockedout.csv)
Example data, but I had to redact it heavily (including partition keys, as they contain sensitive information).

**Comment by MRayermannMSFT** [comment]
@bojned we will try repro'ing with your sample data. Thank you for providing that. In the meantime you find 1.21.3 here to unblock you: https://github.com/microsoft/AzureStorageExplorer/releases

**Comment by JasonYeMSFT** [comment]
@bojned I filled dummy partition keys, row keys and value for the UserId (the only property of GUID type) in the sample data and was able to import it to a table with our 1.22.1 release. The import should work if you are importing in an empty table. If you see an error about invalid value, you need to check the redacted values to see if any of them contains invalid values for Table service.

We do realize that we had a regression for not allowing to overwrite entities during import. We have made a fix and it will be included in our 1.23.0 release. https://github.com/microsoft/AzureStorageExplorer/issues/5387

**Comment by bojned** [comment]
Hi Jason, I think dummy data is not enough in this case... As imports for me fail when CSV file has 30.000+ records, not small like this one provided.

I am using 1.21.3 version atm and I do not face this issue with it, but the latest one is not working right.

**Comment by JasonYeMSFT** [comment]
@bojned At this point, I can only see the problem where colliding entities cannot be imported. It is fixed in 1.23.0 so please try to upgrade to 1.23.0 when it is available. I am going to close this issue as duplicate [5387](https://github.com/microsoft/AzureStorageExplorer/issues/5387). Feel free to create a separate issue for the performance problem for us to work on in the future.
